src/rgw/rgw_main.cc

```diff
@@ -653,6 +653,23 @@ static int process_request(RGWRados *store, RGWREST *rest, RGWRequest *req, RGWC
 
   dout(1) << "====== req done req=" << hex << req << dec << " http_status=" << http_ret << " ======" << dendl;
```

Comment:
This line should be just before return. Technically the request isnt 'done' until there.